core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/EncodingFactory.java

```diff
@@ -81,25 +81,29 @@ public ColumnPageDecoder createDecoder(List<Encoding> encodings, List<ByteBuffer
       metadata.readFields(in);
       SimpleStatsResult stats = PrimitivePageStatsCollector.newInstance(metadata);
       return new AdaptiveIntegralCodec(metadata.getSchemaDataType(), metadata.getStoreDataType(),
-          stats).createDecoder(metadata);
+          stats, encodings.contains(Encoding.INVERTED_INDEX),
```

Comment:
why two times we are calling encodings.contains(Encoding.INVERTED_INDEX) same value can't be use